Makefile.ci

```diff
@@ -0,0 +1,17 @@
+# -*- Makefile -*-
+
+# Usage in Jenkins:
+# make -f Makefile.ci package
+
+.PHONY: package
+package: obs-package-from-git
+	./$^ \
+		-P YaST:Head -p snapper \
+		-o .obsdir \
+		-c 'make -f Makefile.repo && make package'
+
+# A script that is shared with other projects and should be always refreshed
+URL=https://raw.githubusercontent.com/openSUSE/obs-package-from-git/master/obs-package-from-git
+.PHONY: always_out_of_date
+obs-package-from-git: always_out_of_date
+	wget -O $@ $(URL) && chmod +x $@
```

Comment:
but it can even now modify e.g. Makefile.repo, so what is difference?

I think main issue is that if we check on jenkins all pull requests, then we cannot use anything from that git repo, which is not true for any current project, so we have to address it when we decide to add such functionality on jenkins.